### PR TITLE
Fix additional issues

### PR DESCRIPTION
**Pull request by logand22**
This PR fixes all issues with the ecr script.

A successful run can be shown here: https://console.cloud.google.com/cloud-build/builds;region=global/88a981cf-1345-487d-97a9-bee46ae8ab47?project=ci-account

### CODE REVIEW

File: .cloudbuild/publish-image-ecr.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@ STAGING_DOCKER_IMAGE=146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/
 PRODUCTION_DOCKER_IMAGE=public.ecr.aws/gravitational/docs:latest
 
 ## Install aws
-apt install awscli
+apt install -y unzip
```
Comment (tcsc) [inline .cloudbuild/publish-image-ecr.sh:8]: Why the change to AWSCLI v2?

**Review by tcsc — APPROVED** [review]

File: .cloudbuild/publish-image-ecr.sh (hunk 1 above)
Comment (logand22) [reply .cloudbuild/publish-image-ecr.sh:8]: The `ecr-public` subcommand is not available on the default package installed by apt repositories. I've decided to just install based on aws installation guides.
